src/v2/cookbook/cookbook-contributions.md

```diff
@@ -0,0 +1,59 @@
+title: Cookbook Contributions
+type: cookbook
+order: 1000
+---
+
+## What we're looking for
+
+The Cookbook gives developers examples to work off of that both cover common or interesting use cases, and also progressively explain more complex detail. Our goal is to move beyond a simple introductory example, and demonstrate concepts that are more widely applicable, as well as some caveats to the approach. If you're interested in contributing, please follow the template below as much as possible. Some sections are required, and some are optional.
+
+## Simple Example
+
+_required_
+
+- Articulate the problem in a sentence or two.
+- Explain the simplest possible solution in a sentence or two.
+- Show a small code sample.
+- Explain what this accomplishes in a sentence.
+
+## Details about the Value
+
+_required_
+
+- Address common questions that one might have while looking at the example. (Blockquotes are great for this)
+- Show examples of common missteps and how they can be avoided.
+- Show very simple code samples of good and bad patterns.
+- Discuss why this may be a compelling pattern. Links for reference are not required but encouraged.
+
+## Real-World Example
+
+_required_
+
+Demonstrate the code that would power a common or interesting use case, either by:
+- Walking through a few terse examples of setup, or
+- Embedding a codepen/jsfiddle example
+
+If you choose to do the latter, you should still talk through what it is and does.
+
+## Additional Context
+
+_optional_
+
+It's extremely helpful to write a bit about this pattern, where else it would apply, why it works well, and run through a bit of code as you do so or give people further reading materials here.
+
+## When To Avoid This Pattern
+
+_optional_
+
+This section is not required, but heavily recommended. It won't make sense to write it for something very simple such as toggling classes based on state change, but for more advanced patterns like mixins it's incredibly vital. The answer to most questions about development is ["It depends!"](https://codepen.io/rachsmith/pen/YweZbG). This section should take an honest look at when the pattern is useful and when it should be avoided, or when something else makes more sense.
+
+## Alternative Patterns
+
+_optional, except when the section above is provided_
+
+This section is required when you've provided the section above about avoidance. It's important to explore other methods so that people told that something is an antipattern in certain situations are not left high and dry. In doing so, consider that the web is a big tent and that many people have different codebase structures and are solving different goals. Is the app large or small? Are they integrating Vue into an existing project, or are they building from scratch? Are their users only trying to achieve one goal or many? Is there a lot of asyncronous data? All of these concerns will impact alternative implementations. A good cookbook recipe gives developers this context.
```

Comment:
I think "left high and dry" might be difficult for non-native speakers to understand. How about "left wondering"?

Reply:
Good call! I need to get in this mindset of internationalization